feat: Device count via deviceauth-enterprise cli. (#1948)
mendersoftware/mender-docs: 07.Server-installation/04.Production-installation-with-kubernetes/06.Enabling-device-license-count/docs.md

---
title: Enabling device license count
taxonomy:
    category: docs
    label: tutorial
---

> 

Reviewer: Please add a disclaimer:

```
!!!!! Device License Count is only available in Mender Enterprise.
!!!!! Ignore this section if you are running Mender Open Source.
```

Author: added.

> In order to enable the [device license count](../../07.Device-license-count/docs.md) in your cluster,
> you need to schedule a cronjob to run the device-count command every day. If using the helm chart
> installation, the cronjob is already prepared and will be running every day
> <!--AUTOVERSION: "at [3am](https://github.com/mendersoftware/mender-helm/blob/%/mender/templates/device-auth-cron-license-count.yaml)."/ignore-->
> at [3am](https://github.com/mendersoftware/mender-helm/blob/master/mender/templates/device-auth-cron-license-count.yaml).

Reviewer: ```suggestion
To enable the [device license count](../../07.Device-license-count/docs.md) in your cluster,
you must schedule a cronjob to run the device-count command daily. If you are using the Helm chart
installation, the cronjob is already prepared and will be running daily
<!--AUTOVERSION: "at [3am](https://github.com/mendersoftware/mender-helm/blob/%/mender/templates/device-auth-cron-license-count.yaml)."/ignore-->
at [3am](https://github.com/mendersoftware/mender-helm/blob/master/mender/templates/device-auth-cron-license-count.yaml).
```

Author: sure.

> In a similar way if you are running Mender without the helm chart you have to configure

Reviewer: ```suggestion
Similarly, if you are running Mender without the Helm chart, you must configure
```

Author: sure.
